### PR TITLE
solved homework from 1 to 3 challenges

### PR DESCRIPTION
**Pull request by wonmaungthein**
I will continue other homework as soon as possible.

### CODE REVIEW

File: function.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    console.log ("I am " + name + " who is " + age + " years old." + "I am a " 
+    + occupation + " and currently living in " + place + ".");
+}
+
+// we are calling the function and 
+// passing it the data it needs
+aboutMe("Won Maung Thein", 33, "student", "Edinburgh");
+
+
+myClass(students,variety,)
```
Comment (willwright82) [inline function.js]: I know you are only testing things and taking notes here, but be careful with uncommented code that doesn't work, as that can give you confusing errors later.

Comment (wonmaungthein) [inline function.js]: Thank you so much Will.. I will edit it again..

File: homework/challenge01.js [hunk 2]
```diff
@@ -1,41 +1,117 @@
...
+
+if (alive === true) {
+    console.log(alive);
+}
+else {
+  console.log(notAlive);
+}
+
+
+alive ("firstName,lastName,age,occupation") {
```
Comment (willwright82) [inline homework/challenge01.js]: You need to declare a function here, something like this:
```
function findAlive(writers) {
  for (var i = 0, len = writers.length; i < len; i++) {
    //Check if each writer is alive and console.log it if true
  }
}
```

File: homework1/challenge03.js [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+Should return:
+'Found the needle at position 2'
+*/
+
+function findNeedle(haystack) {
+    var result = "Found the needle at position " + i;
+    for (var i = 0; i < haystack.length; i++) {
+        if (haystack[i] === "needle") {
+            result = "Found the needle at position " + i;
+        }
```
Comment (willwright82) [inline homework1/challenge03.js]: Just a note, you could also add an `else` here and say "No needle found"

File: homework1/challenge05.js [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+/* Challenge 5: Ones and Zeros
+Write a function `stringy` that takes a size and returns a string of alternating '1s' and '0s'.
+The string should start with a 1.
+```js
+stringy(6) // --> '101010'
+stringy(5) // --> '10101'
+```
+The size will always be positive and will only use whole numbers.
+*/
+var number = 5;
```
Comment (willwright82) [inline homework1/challenge05.js]: Try and use a better variable name here… number doesn't mean much

File: homework1/challenge05.js [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+
+function stringy(number) {
+    var string = " ";
+        for (var i = 0; i < number.length; i++) {
```
Comment (willwright82) [inline homework1/challenge05.js]: This gives an error:
```
5.length
^^
SyntaxError: Invalid or unexpected token
```
You want the loop to happen `number` times (5 times), so just use 
```
for (var i = 0; i < number; i++) {
```

File: homework1/challenge05.js [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+}
+
+console.log(stringy(number));
+
+var bigNumber = 6;
+console.log(stringy(bigNumber));
+
+function stringy(number) {
+    var result = "";
+    for (var i = 0; i < number; i++) {
```
Comment (willwright82) [inline homework1/challenge05.js]: I see you fixed it here, but remember to keep your code DRY (**D**on't **R**epeat **Y**ourself)…

File: homework1/challenge06.js [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+removeVowels(alphabet) // --> ['b', 'c', 'd', 'f', ... 'y', 'z']
+// */
+
+var alphabet = ['a', 'b', 'c', 'd', 'e',
+    'f', 'g', 'h', 'i', 'j', 'k', 'l', 'm', 'n',
+    'o', 'p', 'q', 'r', 's', 't', 'u', 'v', 'w', 'x', 'y', 'z'];
+var vowels = ['a', 'e', 'i', 'o', 'u'];
+
+function removeVowels() {
+    var array = [];
```
Comment (willwright82) [inline homework1/challenge06.js]: Just don't call an array `array` — this could be `solutionArray` for instance

**Pull request closed without merging**